pkg/auth/login/login.go

```diff
@@ -44,12 +44,12 @@ func (a *AuthorizationCodeGrant) Execute(ctx context.Context, ssoCfg *SSOConfig,
 	}
 	a.Logger.Info(a.Localizer.MustLocalize("login.log.info.loggedIn"))
 
-	a.Logger.Info(a.Localizer.MustLocalize("login.log.info.loggingInMAS"))
+	// a.Logger.Info(a.Localizer.MustLocalize("login.log.info.loggingInMAS"))
 	// log in to MAS-SSO
-	if err := a.loginMAS(ctx, masSSOCfg); err != nil {
-		return err
-	}
-	a.Logger.Info(a.Localizer.MustLocalize("login.log.info.loggedInMAS"))
+	// if err := a.loginMAS(ctx, masSSOCfg); err != nil {
+	// 	return err
+	// }
+	// a.Logger.Info(a.Localizer.MustLocalize("login.log.info.loggedInMAS"))
```

Comment:
Did you try logging in with an offline token from https://cloud.redhat.com/openshift/token?

```shell
rhoas login --token=" ... "
```

When logging in this way, it will not try to log in to MAS-SSO at all, so the above commented out block won't be hit, and might not need to be commented out.